packages/core-mobile/app/navigation/WalletScreenStack/WalletScreenStack.tsx

```diff
@@ -240,177 +241,180 @@ function WalletScreenStack(props: Props): JSX.Element {
     <BridgeProvider>
       <NFTMetadataProvider>
         {isTestnet && <TestnetBanner />}
-        <WalletScreenS.Navigator
-          screenOptions={{
-            headerShown: false
-          }}>
-          <WalletScreenS.Screen
-            name={AppNavigation.Wallet.Drawer}
-            component={DrawerScreenStack}
-          />
-          <WalletScreenS.Screen
-            options={{
-              ...MainHeaderOptions({ title: 'Manage token list' })
-            }}
-            name={AppNavigation.Wallet.TokenManagement}
-            component={TokenManagement}
-          />
-          <WalletScreenS.Screen
```

Comment:
when toggling testnet mode, the animation/transition is kinda glitchy
https://github.com/user-attachments/assets/b4588b9b-a146-4595-85b3-6b22adac4f9f

Reply:
vs the current core mobile
https://github.com/user-attachments/assets/2999327b-2508-4c7b-9074-daab566fba5c

Reply:
This wasn't an easy fix, but I was able to figure out something.